### ISSUE TITLE
Workspace removed in 4.1.1 even with shallowClone: false

### ISSUE THREAD

**Issue by superbiche** [shipitjs/shipit#212]
Hi there,

First thanks for offering me a smooth way to leave Capistrano and use JS :)  

The https://github.com/shipitjs/shipit/releases/tag/v4.1.1 release - specifically shipit-deploy - doesn't seem to implement https://github.com/shipitjs/shipit/commit/6ba6f0077b5a677aaf5b27bcffb419e142f67178 properly, at least in the lib task:  

### Config
```javascript
shipit.initConfig({
    default: {
      repositoryUrl: process.env.DEPLOY_REPO_URL,
      workspace: process.env.DEPLOY_WORKSPACE,
      branch: 'master',
      keepReleases: 5,
      // shallowClone is correctly set to false
      shallowClone: false, 
      ignores: [
        '.env',
        'node_modules',
        'node/node_modules',
        'shipitfile.js',
        'bootstrap/cache',
        'public/.htaccess',
        'public/storage',
        'public/robots.txt',
        'storage/app',
        'storage/logs',
      ],
      shared: {
        overwrite: true,
        dirs: [
          'bootstrap/cache',
          'storage/app',
          'storage/logs',
        ],
        files: [
          '.env',
          'public/.htaccess',
          'public/robots.txt',
        ]
      }
    },
    staging: {
      branch: 'master',
      deployTo: process.env.DEPLOY_STAGING_DEPLOY_TO,
      servers: {
        user: process.env.DEPLOY_STAGING_SSH_USER,
        host: process.env.DEPLOY_STAGING_SSH_HOST,
        port: process.env.DEPLOY_STAGING_SSH_PORT
      },
      keepReleases: 1,
    },
  })
```

### Output
```Revision file created.  
Removing workspace "/Users/michel/tmp/deploy/dentalsirh"
Workspace removed.
Finished 'deploy:update' after 15 s  
```

### `node_modules/shipit-deploy/lib/tasks/deploy/update.js`
```javascript
 let removeWorkspace = (() => {
      var _ref8 = _asyncToGenerator(function* () {
        // I guess the condition should be here, but it isn't.
        shipit.log(`Removing workspace "${shipit.workspace}"`);
        yield (0, _rmfr2.default)(shipit.workspace);
        shipit.log(_chalk2.default.green('Workspace removed.'));
      });

      return function removeWorkspace() {
        return _ref8.apply(this, arguments);
      };
    })();

    (0, _extendShipit2.default)(shipit);

    yield setPreviousRelease();
    yield setPreviousRevision();
    yield createReleasePath();
    yield copyPreviousRelease();
    yield remoteCopy();
    yield setCurrentRevision();
    yield removeWorkspace();
```

Also I don't know if you're looking for contributors to help or not, but I'm totally in if so.

Cheers,

**Comment by neoziro** [comment]
Hi @superbiche, yes I am always looking for help for Shipit! If you can fix it, please submit a PR!

**Comment by superbiche** [comment]
Hi @neoziro, great, I'm back from vacation, just have a few emergencies to tackle then I'll PR the fix.

**Comment by btkostner** [comment]
Should be fixed in #200

**Comment by superbiche** [comment]
@btkostner indeed but there was no version bump since your fix was merged!
@neoziro could you publish a new version? Thanks
